### ISSUE TITLE
Seadrive doesn't update server when file timestamp changed

### ISSUE THREAD

**Issue by munrobasher** [haiwen/seafile#2775]
If you change just the timestamp on a file using PowerShell like this:

```
$Now = Get-Date
(Get-Item $Path).LastWriteTime = $Now
(Get-Item $Path).CreationTime= $Now

```

The changes are **not** uploaded to the server by Seadrive. However, if you do the same on a computer running Seafile client, the timestamps **are** updated on the server. In addition, any updates made via Seafile client on one computer are picked up by Seadrive on another computer.

**Comment by munrobasher** [comment]
Additional information, when the above PowerShell is run on a file manager by Seadrive, the icon **does** change to synchronising but it never actually synchronises.

![SimpleHelp_Technician_LobaWwrX5h](https://github.com/haiwen/seafile/assets/13134946/02e196b8-67e8-405f-9954-4c24836e42fa)

**Comment by killing** [comment]
Placeholder files are not tracked. So any changes to their metadata is not uploaded to the server. I think this is not a very common use case.

**Comment by munrobasher** [comment]
I agree it's not a common scenario but Seafile client and Seadrive work differently. They should at least be consistent.
